Phonebook.Frontend/src/app/pages/dashboard/components/team/team.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="pb-team">
+<div class="different-cards">
+  <button class="navigation-button" mat-button [routerLink]="['/dashboard/team']">
+    <div class="card-navigation">
+    <mat-icon>people</mat-icon>
+    <span class="navigation-name" i18n="DashboardComponent|Title of the Team people section@@DashboardComponentTitleTeam">
+      My Team
+    </span>
+  </div>
+  </button>
+  </div>
+<div cdkDropListGroup class="pb-team-list pb-flex-row  pb-overflow-auto">
+  <div
+    *ngFor="let teamPerson of teamPersons"
+    class="card-container"
+  >
+    <app-user-small-card
+      [person]="teamPerson"
+      [actionButtonClasses]="'white'"
+      class="pb-small-card pb-card"
+    ></app-user-small-card>
+  </div>
+</div>
+<div class="different-cards bottom-button">
```

Comment:
Please move the navigation to the dashboard component. 
Why? This way we can reuse the teams component in other parts of the application.